packages/firestore/test/unit/specs/spec_test_runner.ts

```diff
@@ -1360,8 +1367,10 @@ export interface SpecStep {
   expectedSnapshotsInSyncEvents?: number;
 }
 
-/** [<target-id>, <query-path>] */
-export type SpecUserListen = [TargetId, string | SpecQuery];
+export interface SpecUserListen {
```

Comment:
HOLY SMOKES. THIS IS THE BEST THING I HAVE READ ALL YEAR.

Thanks for cleaning this up.